### PR TITLE
Flyio/setup

### PR DESCRIPTION
**Pull request by brueland**
Setup and config needed to deploy to Fly.io at bonotebo.fly.dev

Dockerfile Image version pinned at python3.12 (no slim), needed to install high enough Nodejs/NPM version for tailwind 

needed tailwind install/build 

static root and static url set to "static/" serves both tailwind theme and default admin css

### CODE REVIEW

File: requirements.txt
Comment (theandrew168) [inline requirements.txt:1]: This file is still showing up as that Windows UTF16 format. I wonder if there is a way to fix that from the Windows side?

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# DB
+*.sqlite3
+
+# Virtual Envs
+.venv/
+venv/
+
+# Static Files
+static/
```
Comment (theandrew168) [inline .dockerignore]: I think we should add `theme/static/` here, too. Tailwind places its output there and we should probs keep that out of the image.

File: .gitignore [hunk 2]
```diff
@@ -164,3 +164,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Static Files
+static/
```
Comment (theandrew168) [inline .gitignore:169]: Same deal here: I think we should add `theme/static/` here, too. Tailwind places its output there and we should probs keep that out of the repo.

**Review by theandrew168 — APPROVED** [review]
Sickkkkkk. We won this battle today. We'll win the next one, too.

File: requirements.txt
Comment (brueland) [reply requirements.txt:1]: What was the command you used to fix it yesterday?

Comment (theandrew168) [reply requirements.txt:1]: I used [dos2unix](https://dos2unix.sourceforge.io/). Its super easy to install on *nix systems but I'm not sure about Windows. You can always merge this as-is and I'll fix it in a follow-up.

**Pull request merged**